example_data/sample_bundle.json

```diff
@@ -34,72 +49,75 @@
         ]
       },
       "request": {
-        "method": "POST",
-        "url": "Organization"
+        "method": "PUT",
+        "url": "Organization?_id=gosh-redcap-20RG-205G0047"
       }
     },
     {
-      "fullUrl": "Patient/37176e84-d977-e993-3c49-d76fcfc6e625",
+      "fullUrl": "Patient/gosh-redcap-40388914",
       "resource": {
         "resourceType": "Patient",
-        "id": "37176e84-d977-e993-3c49-d76fcfc6e625",
+        "id": "gosh-redcap-40388914",
         "active": true,
         "name": [
           {
-            "family": "Manager",
+            "family": "BrighGosh321",
             "given": [
-              "General"
+              "Maria321"
             ]
           }
         ],
-        "gender": "female",
-        "birthDate": "2016-06-01",
-        "managingOrganization": {
-          "reference": "Organization/85940927-468f-f53d-864a-7a50b48d73f1"
+        "gender": "male",
+        "birthDate": "2019-08-02",
+        "text": {
+          "status": "generated",
+          "div": "<div xmlns=\\\"http://www.w3.org/1999/xhtml\\\">Maria321 BrighGosh321 from RedCap FHIR</div>"
         },
         "identifier": [
           {
-            "value": "40057119"
+            "use": "official",
+            "assigner": {
+              "display": "GOSH MRN"
+            },
+            "system": "https://fhir.nhs.uk/Id/mrn",
+            "value": "40388914"
           },
           {
-            "extension": [
-              {
-                "url": "https://fhir.nhs.uk/R4/StructureDefinition/Extension-UKCore-NHSNumberVerificationStatus",
-                "valueCodeableConcept": {
-                  "coding": [
-                    {
-                      "system": "https://fhir.nhs.uk/R4/CodeSystem/UKCore-NHSNumberVerificationStatus",
-                      "code": "Z60791509",
-                      "display": "Family Number"
-                    }
-                  ]
-                }
-              }
-            ]
+            "system": "https://fhir.nhs.uk/Id/nhs-number",
+            "value": "8105688202.0"
+          },
+          {
+            "system": "https://fhir.nhs.uk/Id/nhs-family-number",
```

Comment:
nhs number is fine, but probably should create our own systems in FSH and then use those as the system? What do you think - I'm happy to implement this if you think it's a good idea

Reply:
Let's implement our own in FSH. It's better.